CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 - Sync `connectionTimeout` and `readTimeout` to Android ([#1397](https://github.com/getsentry/sentry-dart/pull/1397))
 - Set User `name` and `geo` in native plugins ([#1393](https://github.com/getsentry/sentry-dart/pull/1393))
 - Add processor count to device info ([#1402](https://github.com/getsentry/sentry-dart/pull/1402))
+- Add attachments to `Hint` ([#1404](https://github.com/getsentry/sentry-dart/pull/1404))
```

Comment:
```suggestion
- Add attachments to `Hint` ([#1404](https://github.com/getsentry/sentry-dart/pull/1404))

```dart
    final text = 'This event should not be sent happen in prod. Investigate.';
    final textAttachment = SentryAttachment.fromIntList(
      utf8.encode(text),
      'event_info.txt',
      contentType: 'text/plain',
    );
    await Sentry.captureException(
      exception,
      stackTrace: stacktrace,
      hint: Hint.withAttachment(textAttachment),
    );
```
```

Reply:
Indentation is probably wrong and the backticks in suggestion does not work, but you got the idea.

Reply:
@denrase the snippet is fine, just remove the first 2 spaces of each line.